app/models/openscap_result.rb

```diff
@@ -0,0 +1,68 @@
+require 'openscap'
+require 'openscap/ds/arf'
+require 'openscap/xccdf/benchmark'
+
+class OpenscapResult < ApplicationRecord
+  belongs_to :container_image
+  has_one    :binary_blob,           :dependent => :destroy, :autosave => true, :as => :resource
+  has_many   :openscap_rule_results, :dependent => :destroy, :autosave => true
+
+  OpenSCAP.oscap_init
```

Comment:
I would suggest we do something similar to `FileDepotFtp#with_connection` method.  Maybe something like the following:
```ruby
def with_open_scap
  raise "no block given" unless block_given?
     begin
      OpenSCAP.oscap_init
      yield
    ensure
      OpenSCAP.oscap_cleanup
    end
end
```